octobot_trading/exchanges/exchange_factory.py

```diff
@@ -99,6 +104,24 @@ async def _initialize_exchange_backend(exchange_manager):
         except Exception as e:
             exchange_manager.is_valid_account = False
             exchange_manager.logger.exception(e, True, f"Error when loading exchange account: {e}")
+        finally:
+            if exchange_manager.is_valid_account:
+                exchange_manager.logger.info("On behalf of the OctoBot team, thank you for "
+                                             "actively supporting the project !")
+
+
+async def _is_supporting(exchange_manager) -> bool:
+    if exchange_manager.community_authenticator is None:
+        return False
+    try:
+        if not exchange_manager.community_authenticator.is_initialized():
+            initialization_timeout = 5
```

Comment:
The exchange creation is delayed by max 5s if the community website doesn't respond right ?

Reply:
yes, that's the idea